kubectl-plugin/uninstall/component/servicemesh.go

```diff
@@ -0,0 +1,40 @@
+package component
+
+import (
+	"github.com/kubesphere-sigs/ks/kubectl-plugin/common"
+)
+
+// ServiceMesh return the struct of ServiceMesh
+type ServiceMesh struct {
+}
+
+// Uninstall uninstall ServiceMesh
+func (s *ServiceMesh) Uninstall() error {
+	if err := common.ExecCommand("wget", "https://github.com/istio/istio/releases/download/1.12.1/istioctl-1.12.1-linux-amd64.tar.gz"); err != nil {
+		return err
+	}
+	if err := common.ExecCommand("tar", "-xvf", "istioctl-1.12.1-linux-amd64.tar.gz"); err != nil {
+		return err
+	}
+	if err := common.ExecCommand("cp", "istioctl", "/usr/local/bin"); err != nil {
+		return err
+	}
```

Comment:
You can install `istioctl` via the following code:

```golang
is := installer.Installer{
    Provider: "github",
}
if err = is.CheckDepAndInstall(map[string]string{
    "istioctl": "istio/istio",
}); err != nil {
    return
}
```

See also https://github.com/LinuxSuRen/hd-home/pull/143

Please see my manual test output:
```shell
$ hd install istio
start to fetch the config
target package is istio/istio
start to download from https://gh.api.99988866.xyz/github.com/istio/istio/releases/download/1.12.1/istioctl-1.12.1-linux-amd64.tar.gz
start to download with 8 threads, size: 21749232, unit: 2718654
Downloading part 7:  [====================================================================] 100%
Downloading part 4:  [====================================================================] 100%
Downloading part 2:  [====================================================================] 100%
Downloading part 5:  [====================================================================] 100%
Downloading part 6:  [====================================================================] 100%
Downloading part 1:  [====================================================================] 100%
Downloading part 3:  [====================================================================] 100%
Downloading part 0:  [====================================================================] 100%
install ./istioctl to /usr/local/bin/istioctl
```

Reply:
I had got some errors like this :
`
 error   installer       failed to list IstioOperator CR: no matches for kind "IstioOperator" in version "install.istio.io/v1alpha1"
All Istio resources will be pruned from the cluster
Proceed? (y/N) failed to get proxy infos: unable to find any Istiod instances
`
and the version of `http-downloader` dependency package is `0.0.55`
```
github.com/linuxsuren/http-downloader v0.0.55
```